.travis.yml

```diff
@@ -14,3 +14,7 @@ install:
 
 script:
   - python setup.py test
+
+notifications:
+  slack:
+    secure: k8OosA/bLxoPdaIL3tsli3OZkqXCkK/eOtGLTX79KslyvzFalwmKwtzyvGLus7mOjo7YAmjFi3vcltHKNseQzhw2gcclOAPNZCHPWEt4VtE7rsd3fWy5JlVnJ5jMN9dkZZ4TJOCQCydBodjq9HA/SLLPfP7IBVddUx4lu29WnEpcms0WKSk/XyWLGPGz09VwYZRr7S/C6wm+vK6zda66H+azkFtdLR2mmPeg8bCsAveBhAYOIpNF5dGhigFtT5xEjEXEbkaKWAt3iTooN0QsqaeRtTMyawc6Ybb+VxKoSCWtZKkflr6dnQA7xr3pkzMVJwS1PJCvtVWlIdCml6g2jkjT2td6jTIg5DSApoH/9f+qAI/9pkK3BV4gMm+jgk7B2Q3wPwWVTa5L19Zj6EpZU8k5H6uz1Da7uByTxZTBD94luQ2KcpPRmFjzyFQUYqLg70DpAWUJhqb3hDDjZcA19Jx6T7xjzubwYj9yFqYQde6Nmpnd7gCnyVHiRJ9DbTPvgws8MxCOSlDQSEKJImNYfe2rNT4lrMFfBX9EP4Bq0PlADNNctLIpQJ4kn28h3nO5wSg+GSOs2sKeCndrp91BR6d+ai7XgoDsSce+HNJWc9/z9D2wfzlgoa03blVNWuEtlERjisEV2gDpdscUgbrE/1wgEPzIFCX01MXX9HIMAGc=
```

Comment:
This is encrypted and tied between this repo and a specific room, so would be useless to anyone else